### PR TITLE
Add mappings for inventory-related

### PR DESCRIPTION
**Pull request by enbrain**

### CODE REVIEW

File: mappings/net/minecraft/inventory/Inventories.mapping [hunk 1]
```diff
@@ -1,4 +1,14 @@
 CLASS net/minecraft/class_1262 net/minecraft/inventory/Inventories
+	METHOD method_29234 clear (Lnet/minecraft/class_1263;Ljava/util/function/Predicate;IZ)I
+		ARG 0 inventory
+		ARG 1 shouldRemove
+		ARG 2 maxCount
+		ARG 3 dryRun
+	METHOD method_29235 clear (Lnet/minecraft/class_1799;Ljava/util/function/Predicate;IZ)I
+		ARG 0 itemStack
```
Comment (liach) [inline mappings/net/minecraft/inventory/Inventories.mapping]:
```suggestion
		ARG 0 stack
```

**Review by liach — APPROVED** [review]

**Comment by shartte** [comment]
Is `clear` really right here? Not `remove`?
Clear suggest well, everything being removed.

**Comment by enbrain** [comment]
I named it `clear` just because it is used by `ClearCommand`.
`remove` looks better.

File: mappings/net/minecraft/inventory/Inventories.mapping [hunk 2]
```diff
@@ -1,4 +1,14 @@
 CLASS net/minecraft/class_1262 net/minecraft/inventory/Inventories
+	METHOD method_29234 remove (Lnet/minecraft/class_1263;Ljava/util/function/Predicate;IZ)I
+		ARG 0 inventory
+		ARG 1 shouldRemove
+		ARG 2 maxCount
+		ARG 3 dryRun
```
Comment (i509VCB) [inline mappings/net/minecraft/inventory/Inventories.mapping:8]: I have no idea what `dryRun` does. Javadoc would be nice on the parameter

**Review by shartte — APPROVED** [review]

File: mappings/net/minecraft/inventory/Inventories.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/inventory/Inventories.mapping:8]: Hmm, don't people know that a dry run is a simulation run? I am surprised that you haven't heard of the term.

File: mappings/net/minecraft/inventory/Inventories.mapping [hunk 3]
```diff
@@ -1,4 +1,18 @@
 CLASS net/minecraft/class_1262 net/minecraft/inventory/Inventories
+	METHOD method_29234 remove (Lnet/minecraft/class_1263;Ljava/util/function/Predicate;IZ)I
+		COMMENT @return the number of items which got removed
+		ARG 0 inventory
+		ARG 1 shouldRemove
+		ARG 2 maxCount
+		ARG 3 dryRun
+			COMMENT If set to true, it returns the number of items which satisfy the condition without actually removing them.
```
Comment (liach) [inline mappings/net/minecraft/inventory/Inventories.mapping]:
```suggestion
			COMMENT whether to return the number of items which would have been removed without actually removing them
```

File: mappings/net/minecraft/inventory/Inventories.mapping [hunk 4]
```diff
@@ -1,4 +1,18 @@
 CLASS net/minecraft/class_1262 net/minecraft/inventory/Inventories
+	METHOD method_29234 remove (Lnet/minecraft/class_1263;Ljava/util/function/Predicate;IZ)I
+		COMMENT @return the number of items which got removed
```
Comment (liach) [inline mappings/net/minecraft/inventory/Inventories.mapping]:
```suggestion
		COMMENT Removes a number, not exceeding {@code maxCount}, of items from an inventory based on a predicate and returns that number.
		COMMENT @return the number of items removed
```

File: mappings/net/minecraft/inventory/Inventories.mapping [hunk 5]
```diff
@@ -1,4 +1,18 @@
...
+		ARG 0 inventory
+		ARG 1 shouldRemove
+		ARG 2 maxCount
+		ARG 3 dryRun
+			COMMENT If set to true, it returns the number of items which satisfy the condition without actually removing them.
+	METHOD method_29235 remove (Lnet/minecraft/class_1799;Ljava/util/function/Predicate;IZ)I
+		COMMENT @return the number of items which got removed
```
Comment (liach) [inline mappings/net/minecraft/inventory/Inventories.mapping]:
```suggestion
		COMMENT Removes a number, not exceeding {@code maxCount}, of items from an item stack based on a predicate and returns that number.
		COMMENT @return the number of items removed
```

File: mappings/net/minecraft/inventory/Inventories.mapping [hunk 6]
```diff
@@ -1,4 +1,18 @@
...
+		ARG 0 stack
+		ARG 1 shouldRemove
+		ARG 2 maxCount
+		ARG 3 dryRun
+			COMMENT dryRun If set to true, it returns the number of items which satisfy the condition without actually removing them.
```
Comment (liach) [inline mappings/net/minecraft/inventory/Inventories.mapping]:
```suggestion
			COMMENT whether to return the number of items which would have been removed without actually removing them
```

File: mappings/net/minecraft/inventory/Inventories.mapping (hunk 2 above)
Comment (i509VCB) [reply mappings/net/minecraft/inventory/Inventories.mapping:8]: I know the concept, just not what it does in this context

**Review by liach — APPROVED** [review]

**Review by i509VCB — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**